pkgs/tools/text/difftastic/default.nix

```
@@ -14,14 +13,14 @@ let
   };
 in
 
-rustPlatform.buildRustPackage rec {
+rustPlatform.buildRustPackage (finalAttrs: {
   pname = "difftastic";
   version = "0.46.0";
 
   src = fetchFromGitHub {
     owner = "wilfred";
-    repo = pname;
-    rev = version;
+    repo = finalAttrs.pname;
```

```suggestion
    repo = "difftastic";
```
this is a bit problematic because if someone appends something to pname like ``-unwrapped`` than the download link silently breaks because of FODs.

---

It's worse: it wouldn't even break for you but for someone else, because - having the output - you don't need to re-run the FOD.

D.R.Y: do repeat yourself ;)